lib/app.js

```diff
@@ -0,0 +1,28 @@
+// import dependencies
+import express from 'express';
+import cors from 'cors';
+import { creatures } from '../data/creatures.js';
+
+// make an express app
+const app = express();
+
+// allow our server to be called from any website
+app.use(cors());
+
+// example route to see if app is alive!
+app.get('/', (req, res) => {
+  // send back a short response
+  res.send('Hello World!');
+});
+
+app.get('/api/creatures', (req, res) => {
+  // send back a short response
+  res.json(creatures);
+});
+
+app.get('/api/creatures/:id', (req, res) => {
```

Comment:
again :id is a variable. could be /one, /1, /whatever, you just want to update the variable in your test to show it works.